content/en/agent/guide/private-link.md

```diff
@@ -0,0 +1,147 @@
+
+{{% /tab %}}
+{{% tab "Logs" %}}
+
+| Datadog Log Service Name                                  |
+| --------------------------------------------------------- |
+| `com.amazonaws.vpce.us-east-1.vpce-svc-0a2aef8496ee043bf` |
+
+{{% /tab %}}
+{{% tab "API" %}}
+
+| Datadog API Service Name                                  |
+| --------------------------------------------------------- |
+| `com.amazonaws.vpce.us-east-1.vpce-svc-02a4a57bc703929a0` |
+
+{{% /tab %}}
+{{< /tabs >}}
+
+4. Hit the _verify_ button. If it does not return _Service name found_, reach out to the [Datadog support team][2].
+5. Choose the VPC and subnets that should be peered with the Datadog VPC service endpoint.
+6. Make sure that for **Enable DNS name** the _Enable for this endpoint_ is checked:
+   {{< img src="agent/guide/private_link/enabled_dns_private.png" alt="Enable DNS private" style="width:60%;" >}}
+7. Choose the security group of your choice to control what can send traffic to this VPC endpoint.
+
+    **Note**: **If you want to forward logs to Datadog through this VPC endpoint, the security group must accept inbound and outbound traffic on port `443`**.
+
+8. Hit **Create endpoint** at the bottom of the screen. If successful, you will see this:
+   {{< img src="agent/guide/private_link/vpc_endpoint_created.png" alt="VPC endpoint created" style="width:60%;" >}}
+9. Click on the VPC endpoint ID to check its status.
+10. Wait for the status to move from _Pending_ to _Available_. This can take up to 10 minutes.
+    {{< img src="agent/guide/private_link/vpc_status.png" alt="VPC status" style="width:60%;" >}}
+
+Once it shows _Available_, the AWS PrivateLink is ready to be used. The next step is to update your Agent configurations with the new target endpoint for your Datadog Agents, Lambda forwarder, and/or other clients shipping data to Datadog.
+
+## Client configuration
+
+Select the tab below to see how to send your metrics and logs to Datadog using this new VPC endpoint, or which new host URL you would need to use for Datadog API:
+
+{{< tabs >}}
+{{% tab "Metrics" %}}
+
+_Available for Agent 6.0+_
+
+To forward your metrics to Datadog using this new VPC endpoint, configure `pvtlink.agent.datadoghq.com` as your new metric destination:
+
+1. Update the `dd_url` parameter in the [Agent `datadog.yaml` configuration file][1]:
+
+    ```yaml
+    dd_url: pvtlink.agent.datadoghq.com
+    ```
+
+2. [Restart your Agent][2] to send metrics to Datadog through AWS PrivateLink.
+
+**Note**: if you are using the container Agent, set the environment variable instead: `DD_DD_URL="pvtlink.agent.datadoghq.com"`
+
+
+[1]: /agent/guide/agent-configuration-files/#agent-main-configuration-file
+[2]: /agent/guide/agent-commands/#restart-the-agent
+{{% /tab %}}
+{{% tab "Logs" %}}
+
+_Available for Agent 6.14+_
+
+To forward your logs to Datadog using this new VPC endpoint, you need to set up `pvtlink.logs.datadoghq.com` as your new log destination:
```

Comment:
nit: consistency: 'you need to set up' -> 'configure', for consistency with the Logs tab.